### PR TITLE
Make @fluidframework/attributor public

### PR DESCRIPTION
**Pull request by Abe27342**
## Description

Renames @fluid-internal/attributor to @fluidframework/attributor and makes the package public. All APIs are still marked alpha, but it's in a good enough state to start getting feedback from partner teams.

### CODE REVIEW

**Comment by Abe27342** [comment]
I may want to let #13344 merge first.

File: PACKAGES.md [hunk 1]
```diff
@@ -93,13 +93,13 @@ The dependencies between layers are enforced by the layer-check command._
...
...-manager)</br>- [@fluid-internal/tree](/packages/dds/tree)</br>- [@fluidframework/container-runtime](/packages/runtime/container-runtime)</br>- [@fluidframework/container-runtime-definitions](/packages/runtime/container-runtime-definitions)</br>- [@fluidframework/datastore](/packages/runtime/datastore)</br>- [@fluidframework/datastore-definitions](/packages/runtime/datastore-definitions)</br>- [@fluidframework/garbage-collector](/packages/runtime/garbage-collector)</br>- [@fluidframework/runtime-definitions](/packages/runtime/runtime-definitions)</br>- [@fluidframework/runtime-utils](/packages/runtime/runtime-utils) | - [Base-Definitions](#Base-Definitions)</br>- [Protocol-Definitions](#Protocol-Definitions)</br>- [Driver-Definitions](#Driver-Definitions)</br>- [Container-Definitions](#Container-Definitions)</br>- [Base-Utils](#Base-Utils)</br>- [Protocol-Utils](#Protocol-Utils)</br>- [Telemetry-Utils](#Telemetry-Utils)</br>- [Driver-Utils](#Driver-Utils)</br>- [Container-Utils](#Container-Utils)</br>&nbsp;</br>&nbsp;</br>&nbsp;</br>&nbsp;</br>&nbsp;</br>&nbsp;</br>&nbsp;</br>&nbsp;</br>&nbsp;</br>&nbsp;</br>&nbsp;</br>&nbsp;</br>&nbsp;</br>&nbsp;</br>&nbsp;</br>&nbsp;</br>&nbsp; |
```
Comment (Abe27342) [inline PACKAGES.md:96]: note: there are a couple changes to this file that are technically unrelated to this PR. It's a generated file that we don't enforce is up-to-date in PR gate currently. I reran the command to generate it. See #12882.

File: packages/framework/attributor/package.json [hunk 2]
```diff
@@ -99,6 +98,7 @@
     "typescript": "~4.5.5"
   },
   "typeValidation": {
+    "disabled": true,
```
Comment (Abe27342) [inline packages/framework/attributor/package.json:101]: we probably just want to turn this on once the API is marked out of @alpha.

**Review by clarenceli-msft — APPROVED** [review]
